google/cloud/spanner_dbapi/connection.py

```diff
@@ -41,28 +52,46 @@ def __handle_update_ddl(self, ddl_statements):
         """
         Run the list of Data Definition Language (DDL) statements on the underlying
         database. Each DDL statement MUST NOT contain a semicolon.
-        Args:
-            ddl_statements: a list of DDL statements, each without a semicolon.
-        Returns:
-            google.api_core.operation.Operation.result()
+
+        :type ddl_statements: list
+        :param ddl_statements: a list of DDL statements, each without a semicolon.
+
+        :rtype: :func:`google.api_core.operation.Operation.result()`
+        :returns: result of the operation
         """
         self._raise_if_closed()
         # Synchronously wait on the operation's completion.
         return self._dbhandle.update_ddl(ddl_statements).result()
 
     def read_snapshot(self):
+        """Returns Data Base snapshot if connection is open.
+
+        :rtype: :class:`Snapshot`
+        :returns: database snapshot.
+        """
         self._raise_if_closed()
         return self._dbhandle.snapshot()
 
     def in_transaction(self, fn, *args, **kwargs):
+        """Runs :func:`spanner_v1.Database.run_in_transaction` inside open connection.
```

Comment:
`Perform a unit of work in a linked Transaction, retrying on abort.`
There is nothing wrong with recycling the existing docstrings, especially those referring to the same method.